mappings/net/minecraft/item/Item.mapping

```diff
@@ -33,10 +33,10 @@ CLASS net/minecraft/class_1792 net/minecraft/item/Item
 		ARG 3 slot
 		ARG 4 clickType
 		ARG 5 player
-		ARG 6 commandItemSlot
+		ARG 6 cursorSlot
```

Comment:
Hmm, should the command item slot be renamed if its usage has expanded?

Reply:
I touched on this in the description of the PR, it felt wrong to change it without talking about it though, so for now I just wanted t o bring it up